src/OSPSuite.R/Services/ContainerTask.cs

```diff
@@ -168,11 +168,11 @@ public IParameter[] AllParametersForSensitivityAnalysisMatching(ISimulation simu
 
       public string[] AllStateVariableParameterPathsIn(IModelCoreSimulation simulation) => AllStateVariableParameterPathsIn(simulation?.Model?.Root);
 
-      public string BaseUnitNameByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound).BaseUnitName();
+      public string BaseUnitNameByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound, Error.CannotGetValueByPathUsingWildCard(path)).BaseUnitName();
 
-      public string DimensionNameByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound).DimensionName();
+      public string DimensionNameByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound, Error.CannotGetValueByPathUsingWildCard(path)).DimensionName();
 
-      public bool IsExplicitFormulaByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound)?.Formula.IsExplicit() ?? false;
+      public bool IsExplicitFormulaByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound) => singleQuantityByPath(simulation, path, throwIfNotFound, Error.CannotGetValueByPathUsingWildCard(path))?.Formula.IsExplicit() ?? false;
```

Comment:
I do not like this passing this stuff everywhere.
Do we need to differentiate between Get and set? What about we just change the message to get/set instead ?

Reply:
@msevestre Fair enough... I have unified the message